### PR TITLE
Update console.md

### PR DESCRIPTION
**Pull request by HoweChen**
Modification on Java Installation in the appendix when installing JDK with such error:

```
Exception in thread "main" javax.net.ssl.SSLException: java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty.
```

### CODE REVIEW

File: docs/manual/console.md [hunk 1]
```diff
@@ -1468,3 +1468,24 @@ $ source /etc/profile
 java -version 
 ```
 
+###The trustAnchors parameter must be non-empty 问题
```
Comment (bxq2011hust) [inline docs/manual/console.md]: The markdown syntax is wrong

Comment (JimmyShi22) [inline docs/manual/console.md]:
```suggestion
### The trustAnchors parameter must be non-empty 问题
```

File: docs/manual/console.md [hunk 2]
```diff
@@ -1468,3 +1468,24 @@ $ source /etc/profile
...
+
+####报错信息：
```
Comment (JimmyShi22) [inline docs/manual/console.md]:
```suggestion
#### 报错信息：
```

File: docs/manual/console.md [hunk 3]
```diff
@@ -1468,3 +1468,24 @@ $ source /etc/profile
...
+```
+Exception in thread "main" javax.net.ssl.SSLException: java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty.
+```
+
+####解决方法：
```
Comment (JimmyShi22) [inline docs/manual/console.md]:
```suggestion
#### 解决方法：
```

File: docs/manual/console.md (hunk 1 above)
Comment (JimmyShi22) [inline docs/manual/console.md]:
```suggestion
### The trustAnchors parameter must be non-empty 问题
```

File: docs/manual/console.md [hunk 4]
```diff
@@ -1468,3 +1468,24 @@ $ source /etc/profile
 java -version 
 ```
 
+###The trustAnchors parameter must be non-empty 问题
+
+####报错信息：
+```
```
Comment (JimmyShi22) [inline docs/manual/console.md]:
```suggestion
``` log
```

File: docs/manual/console.md [hunk 5]
```diff
@@ -1468,3 +1468,24 @@ $ source /etc/profile
...
+Exception in thread "main" javax.net.ssl.SSLException: java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty.
+```
+
+#### 解决方法：
+
+```bash
```
Comment (JimmyShi22) [inline docs/manual/console.md]:
```suggestion
``` bash
```

**Comment by cyjseagull** [comment]
请解决完冲突后，重新提一个PR~

**Comment by cyjseagull** [comment]
因修改了历史commit中敏感信息，导致项目有变动；请克隆FISCO-BCOS-DOC后，重新提交PR

**Pull request closed without merging**